https://github.com/Warzone2100/warzone2100/pull/98

commit 714399bfd5c9effa0a9b1bba8f26b625eadeb1da
Author: Bernhard M. Wiedemann <bwiedemann@suse.de>
Date:   Sun Aug 27 07:07:24 2017 +0200

    Make zip file creation reproducible
    
    by adding -X option to not store extra UNIX timestamps
    and by sorting the recursed input file list
    
    See https://reproducible-builds.org/ for why this is good.

diff --git a/data/Makefile.am b/data/Makefile.am
index bc269123b..0cc29fe9f 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -4,7 +4,7 @@ stamp:
 	touch stamp
 
 %.wz: $(abs_srcdir)/% stamp
-	(cd $(srcdir)/$(notdir $<) && $(ZIP) -r0 $(ZIP_UPDATE) $(abs_builddir)/$@ $(filter-out stamp,$(filter-out $<,$(^:$</%=%))) -x '*svn*' -x '*Makefile*' -x '*.svg' || [ $$? -eq 12 ] && true) # zip returns 12 on "nothing to do"
+	(cd $(srcdir)/$(notdir $<) && $(ZIP) -X -r0 $(ZIP_UPDATE) $(abs_builddir)/$@ $$(find $(filter-out stamp,$(filter-out $<,$(^:$</%=%))) -type f | LC_ALL=C sort) -x '*svn*' -x '*Makefile*' -x '*.svg' || [ $$? -eq 12 ] && true) # zip returns 12 on "nothing to do"
 	$(ZIP) -T $@
 	rm -f stamp
 
